### PR TITLE
Improve conflict resolution

### PR DESCRIPTION
**Pull request by brnsolikyl**
Branch deployment: https://improve-conflict.dev.karrot.world/ - _note: only some features will work here, as it uses `master` branch backend_
Backend PR: https://github.com/karrot-dev/karrot-backend/pull/1243

## What does this PR do?

Some improvements of the conflict resolution feature:

- reframe conflict resolution as membership review, review texts and explanations
- proceed with notification changes described in github issue https://github.com/karrot-dev/karrot-frontend/issues/2556
- add issue count on sidenav

## Checklist

- [ ] added a test, or explain why one is not needed/possible...
- [x] no unrelated changes
- [x] asked someone for a code review
- [x] joined [#karrot:matrix.org](https://matrix.to/#/#karrot:matrix.org)
- [ ] added an entry to CHANGELOG.md (description, pull request link, username(s))
- [x] tried out on a mobile device (does everything work as expected on a smaller screen?)

### CODE REVIEW

**Comment by brnsolikyl** [comment]
Still gotta change the translation strings. Many have the conflict wording in it.

**Review by tiltec — COMMENTED** [review]
Did you check if users can subscribe to the issue chat? Also, I think the option to unsubscribe again is missing?

Changing locale files other than `locale-en.json` will have no effect, will get overwritten from transifex by the next "update locales" PR. If the intention is to keep some of the translated messages while changing the key, you need to do some extra work... don't remember exactly, but some fiddling with `tx push`, but _only translated messages_, otherwise it registers the English fallback as actual translation..... might be safer to not do this and let translators re-translate everything.

File: src/notifications/helpers.js [hunk 1]
```diff
@@ -99,12 +99,12 @@ export function useNotificationHelpers () {
...
-      case 'conflict_resolution_continued':
-      case 'conflict_resolution_continued_about_you':
-      case 'conflict_resolution_decided':
-      case 'conflict_resolution_decided_about_you':
+      case 'membership_review_created':
+      case 'membership_review_created_about_you':
+      case 'membership_review_continued':
+      case 'membership_review_continued_about_you':
+      case 'membership_review_decided':
+      case 'membership_review_decided_about_you':
```
Comment (nicksellen) [inline src/notifications/helpers.js:107]: I need to revert these...

File: test/mockBackend/issues.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ export function generateIssue (params = {}) {
     createdAt: subDays(new Date(), 7 + 6),
     createdBy: null,
     status: 'ongoing',
-    type: 'conflict_resolution',
+    type: 'membership_review',
```
Comment (nicksellen) [inline test/mockBackend/issues.js:27]: Needs revert..

**Comment by nicksellen** [comment]
Ok, a few more reverts completed now, it's _done_ as far as I'm concerned now.

**Comment by nicksellen** [comment]
Ok, tests should pass again now :)

**Review by tiltec — APPROVED** [review]

**Pull request merged**